rllib/tests/test_external_multi_agent_env.py

```diff
@@ -24,29 +24,43 @@ def tearDownClass(cls) -> None:
 
     def test_external_multi_agent_env_complete_episodes(self):
         agents = 4
-        ev = RolloutWorker(
-            env_creator=lambda _: SimpleMultiServing(BasicMultiAgent(agents)),
-            policy_spec=MockPolicy,
-            rollout_fragment_length=40,
-            batch_mode="complete_episodes",
-        )
-        for _ in range(3):
-            batch = ev.sample()
-            self.assertEqual(batch.count, 40)
-            self.assertEqual(len(np.unique(batch[SampleBatch.AGENT_INDEX])), agents)
+        for count_steps_by in ("env_steps", "agent_steps"):
+            ev = RolloutWorker(
+                env_creator=lambda _: SimpleMultiServing(BasicMultiAgent(agents)),
+                policy_spec=MockPolicy,
+                rollout_fragment_length=40,
+                count_steps_by=count_steps_by,
+                batch_mode="complete_episodes",
+            )
+            for _ in range(3):
+                ma_batch = ev.sample()
+                steps = 40 if count_steps_by == "env_steps" else 40 * agents
+                self.assertEqual(ma_batch.count, steps)
+                sample_batch = ma_batch.policy_batches[DEFAULT_POLICY_ID]
+                self.assertEqual(
+                    len(np.unique(sample_batch[SampleBatch.AGENT_INDEX])),
+                    agents,
+                )
 
     def test_external_multi_agent_env_truncate_episodes(self):
         agents = 4
-        ev = RolloutWorker(
-            env_creator=lambda _: SimpleMultiServing(BasicMultiAgent(agents)),
-            policy_spec=MockPolicy,
-            rollout_fragment_length=40,
-            batch_mode="truncate_episodes",
-        )
-        for _ in range(3):
-            batch = ev.sample()
-            self.assertEqual(batch.count, 160)
-            self.assertEqual(len(np.unique(batch[SampleBatch.AGENT_INDEX])), agents)
+        for count_steps_by in ("env_steps", "agent_steps"):
+            ev = RolloutWorker(
```

Comment:
I understand this is not related to this PR, but we should rename ev to worker, since it's a RolloutWorker, not an Env.
what do you think?